crates/uv/src/commands/build_frontend.rs

```diff
@@ -76,6 +80,12 @@ enum Error {
          distribution ending in one of: {1}."
     )]
     InvalidSourceDistExt(String, uv_distribution_filename::ExtensionError),
+    #[error("The built source distribution has an invalid filename")]
+    InvalidBuiltSourceDistFilename(#[source] uv_distribution_filename::SourceDistFilenameError),
+    #[error("The built wheel has an invalid filename")]
+    InvalidBuiltWheelFilename(#[source] uv_distribution_filename::WheelFilenameError),
```

Comment:
Add some extra context because it's not the input filename that's wrong, it's the output filename